python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -240,8 +240,14 @@ def shutdown(self):
         logger = getLogger()
         logger.debug("Shutting Down GangaRepositoryLocal: %s" % self.registry.name)
         for k in self._fully_loaded:
-            self.index_write(k, shutdown=True)
-        self._write_master_cache(True)
+            try:
+                self.index_write(k, shutdown=True)
+            except Exception as err:
```

Comment:
These were failing with KeyError, OSError and others so until the function is safer I'm leaving it as this and moving on. The index by definition doesn't (or at least shouldn't) make a job un-loadable. Yes it's an annoyance if it wasn't writted but it's not worth sacrificing the rest of the shutdown process for it.